dashboard/src/components/ConfigLoader/ConfigLoader.tsx

```diff
@@ -18,7 +18,9 @@ function ConfigLoader({ getConfig, error, ...otherProps }: IConfigLoaderProps) {
   if (error) {
     return <Alert theme="danger">Unable to load Kubeapps configuration: {error.message}</Alert>;
   }
-  return <LoadingWrapper {...otherProps} />;
+  return (
+    <LoadingWrapper className="margin-t-xxl" loadingText={<h1>Kubeapps</h1>} {...otherProps} />
+  );
```

Comment:
> The `LoadingWrapper` from the `Suspense` component didn't appear (so I removed it) and the first one loaded was in the `ConfigLoader` so I added a title the

Oh much better now! Thank you! Minor thing here: since we are allowing modifying Kubeapps by other names, perhaps it should be the following for the sake of consistency:

```suggestion
  return (
    <LoadingWrapper className="margin-t-xxl" loadingText={<h1>{intl.formatMessage({ id: "Kubeapps", defaultMessage: "Kubeapps" })}</h1>} {...otherProps} />
  );
```

Reply:
The ConfigLoader is out of of the IntlProvider:

```
      <ConfigLoaderContainer>
        <ConnectedRouter history={history}>
          <Suspense fallback={null}>
            <IntlProvider
              locale={i18nConfig.locale}
              key={i18nConfig.locale}
              messages={i18nConfig.messages}
              defaultLocale="en"
            >
```

Causing an issue:

```
Error: [React Intl] Could not find required `intl` object. <IntlProvider> needs to exist in the component ancestry.
```

Should we move the IntlProvider up in the tree or leave the ConfigLoader out?

Reply:
Mmm, I don't remember right now, but since we are not using anything from the `ConfigLoaderConainer` in the `IntlProvider` I guess we can move it up.